scheduler.rst

```diff
@@ -105,9 +108,13 @@ The :class:`Symfony\\Component\\Scheduler\\Attribute\\AsSchedule` attribute,
 which by default references the schedule named ``default``, allows you to register
 on a particular schedule::
 
-    // src/Scheduler/MyScheduleProvider.php
+    // src/Scheduler/SaleTaskProvider.php
     namespace App\Scheduler;
 
+    use Symfony\Component\Scheduler\Attribute\AsSchedule;
```

Comment:
How `code-block-checker` can know theses classes ?

Is adding `symfony/scheduler` in its compoer.json will be enought ?

Reply:
Yes this should be enough

Reply:
It's in https://github.com/symfony-tools/symfony-application/pull/26 :see_no_evil:

Reply:
Merged